src/main/java/bridge/model/BridgeLength.java

```diff
@@ -0,0 +1,37 @@
+package bridge.model;
+
+import bridge.util.constant.Errors;
+import bridge.util.constant.Numbers;
+
+import java.util.regex.Pattern;
+
+public class BridgeLength {
+    private final int length;
+
+    public BridgeLength(String length) {
+        this.length = validate(length);
+    }
+
+    private int validate(String length) {
+        //숫자로 이루어져있는 지 구성 → 숫자가 범위 내의 숫자인지
+        return validateRange(isNumber(length));
+    }
+
+    private int isNumber(String length) {
+        if(!Pattern.matches("^[0-9]+$", length)) {
+            throw new IllegalArgumentException(Errors.NOT_NUMBER_ERROR.getMessage());
+        }
+        return Integer.parseInt(length);
+    }
```

Comment:
제가 숫자 변환할 때 자주 쓰는 방법인데 이렇게는 어떨까요?

```suggestion
    private int toInt(String source) {
        try {
            return Integer.parseInt(source);
        } catch (NumberFormatException e) {
            throw new IllegalArgumentException("숫자만 입력 가능합니다.");
        }
    }
```

Reply:
오오 NumberFormatException을 잡아서 IllegelArgumentException으로...! 이건 제가 허를 찔렸네요..! 👍
한번 적용해보겠습니다!